examples/reactnative/__tests__/js/CRUD/create-test.jsx

```diff
@@ -0,0 +1,96 @@
+import React from "react";
+import { Button, TextInput } from "react-native";
+import { render, fireEvent, waitFor, act } from "@testing-library/react-native";
+import Realm from "realm";
+import { createRealmContext } from '@realm/react'
+
+// :snippet-start: crud-dog-schema
+class Dog extends Realm.Object {
+    static schema = {
+        name: "Dog",
+        properties: {
+            name: "string",
+            owner: "Person?",
+            age: "int?",
+        }, 
+    }
+}
+// :snippet-end:
+
+class Person extends Realm.Object {
+    static schema = {
+        name: "Person",
+        properties: {
+            name: "string",
+            age: "int?",
+        },        
+    }
+}
+
+const realmConfig = {
+  schema: [Dog, Person,],
+  inMemory: true,   
+}
+
+const { RealmProvider, useRealm, useObject, useQuery } = createRealmContext(realmConfig);
+
+let assertionRealm;
+
+describe("Create Data Tests", () => {
+    beforeEach(async() => {
+        // we will use this Realm for assertions to access Realm Objects outside of a Functional Component (like required by @realm/react)
+        assertionRealm = await Realm.open(realmConfig)
+
+        // delete every object in the realmConfig in the Realm to make test idempotent
+        assertionRealm.write(()=> {
+            assertionRealm.delete(assertionRealm.objects("Dog"));
+            assertionRealm.delete(assertionRealm.objects("Person"));
+        })
+    })
+    it("should create a new object", async () => {        
+        const wrapper = ({ children }) => <RealmProvider>{children}</RealmProvider>;
+
+        // :snippet-start: crud-create-object
+        // :replace-start: {
+        //  "terms": {
+        //   " testID='handleAddItemBtn'": ""
+        //   }
+        // }
+        const CreateDogInput = () => {
+            const [dogName, onChangeDogName] = React.useState('Fido');
```

Comment:
Typically you would call these [x, setX]
```suggestion
            const [dogName, setDogName] = React.useState('Fido');
```

Reply:
Fixed in the latest commit of https://github.com/mongodb/docs-realm/pull/2381